integrated_channels/degreed/models.py

```diff
@@ -188,6 +192,10 @@ def get_content_metadata_exporter(self, user):
 class DegreedLearnerDataTransmissionAudit(models.Model):
     """
     The payload we sent to Degreed at a given point in time for an enterprise course enrollment.
+
+    .. pii:: The degreed_user_email model field contains PII.
+    .. pii_types:: email_address
+    .. pii_retirement:: consumer_api
```

Comment:
Honestly? I don't understand the difference between `local_api` and `consumer_api`. If the PII is stored in the edx-enterprise models, wouldn't it all be `local_api`? It all needs to be deleted locally, correct?

Reply:
I thought local_api meant that in this codebase there is code to retire the PII.  In this case, the code to retire the PII is in the edx-platform codebase.  Isn't edx-platform the "consumer"?

Reply:
Ah - that makes sense now. The OEP-30 language could use some detail on this point.